policyengine_us/parameters/gov/states/pa/tanf/income/earned_deduction/earned_income_disregard/wed_flat.yaml

```
@@ -0,0 +1,11 @@
+description: Pennsylvania deducts this amount from each adult budget member’s earned income when the individual is eligible for the Earned Income Disregard (EID).
+values:
+  2023-01-01: 2_400 # 200 * 12
```

Multiply by `MONTHS_IN_YEAR` in the Python formula instead (same for others). We represent parameters as close as possible to the raw source.
```suggestion
  2023-01-01: 200
```